src/RoboSaverModule.sol

```diff
@@ -1,16 +1,103 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
-import {IDelayModule} from "./interfaces/modules/IDelayModule.sol";
+import {IERC20} from "@gnosispay-kit/interfaces/IERC20.sol";
+import {IRolesModifier} from "@gnosispay-kit/interfaces/IRolesModifier.sol";
+import {IDelayModifier} from "@gnosispay-kit/interfaces/IDelayModifier.sol";
+
+import {IRoboSaverModule} from "./interfaces/modules/IRoboSaverModule.sol";
 
 contract RoboSaverModule {
```

Comment:
wondering if we should actually call this a module? since it doesnt get installed on the safe but on the delay. it therefore cannot be used as a 'normal' module either. maybe just `RoboSaver`? `RoboSaverGnosisPayModule`? `RoboSaverVirtualModule`?

Reply:
eca3c1bff861a559f881aaa031dc236f0f3df81f